facebook/OpenBIC
# fby3.5: bb: Support re-enable USB power

Summary:
- When inserting debug card into the connector, the debug card might trigger OCP and lead to lose USB power.
- Add to support re-enable USB power to solve USB power lost problem caused by debug card connection.

Test plan:
- Build code: Pass
- Re-enable USB power: Pass

- Log:
1. Insert debug card to check USB power will re-enable.

- Before
  USB power doesn't re-enable and BMC shows SEL.
root@bmc-oob:~# log-util --print all | grep USB
1    slot1    2018-03-09 08:00:05    sensord          ASSERT: Lower Non Critical threshold - raised - FRU: 1, num: 0xE9 curr_val: 0.01 Volts, thresh_val: 4.65 Volts, snr: BB_P5V_USB
1    slot1    2018-03-09 08:00:05    sensord          ASSERT: Lower Critical threshold - raised - FRU: 1, num: 0xE9 curr_val: 0.01 Volts, thresh_val: 4.60 Volts, snr: BB_P5V_USB
1    slot1    2018-03-09 08:00:05    sensord          ASSERT: Lower Non Recoverable threshold - raised - FRU: 1, num: 0xE9 curr_val: 0.01 Volts, thresh_val: 4.15 Volts, snr: BB_P5V_USB

- After
  USB power re-enable and BMC doesn't show wrong SEL.
root@bmc-oob:~# log-util --print all | grep USB
root@bmc-oob:~#
